internal/lookout/application.go

```diff
@@ -6,6 +6,8 @@ import (
 	"strings"
 	"sync"
 
+	"github.com/doug-martin/goqu/v9"
```

Comment:
Should this be `goqu "github.com/doug-martin/goqu/v9"`? For clarity, if nothing else (as-is, I would expect that import to give me a module prefix of `v9`).

Reply:
Apparently `/v{n}` is special and it actually defaults to  `goqu "github.com/doug-martin/goqu/v9"`.